app/lib/frontend/templates/report.dart

```diff
@@ -2,23 +2,34 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+import 'package:pub_dev/frontend/static_files.dart';
+
 import '../../account/models.dart';
 import '../../admin/models.dart';
 import '../dom/dom.dart' as d;
 import '../dom/material.dart' as material;
 import 'layout.dart';
 
 const _subjectKindLabels = {
-  ModerationSubjectKind.package: 'Package: ',
-  ModerationSubjectKind.packageVersion: 'Package version: ',
-  ModerationSubjectKind.publisher: 'Publisher: ',
+  ModerationSubjectKind.package: 'package',
+  ModerationSubjectKind.packageVersion: 'package version',
+  ModerationSubjectKind.publisher: 'publisher',
 };
 
 /// Renders the create publisher page.
 String renderReportPage({
   SessionData? sessionData,
-  ModerationSubject? subject,
+  required ModerationSubject subject,
 }) {
+  final kindLabel = _subjectKindLabels[subject.kind] ?? 'about';
+  // TODO: also add `url`
+  final lcpsDeepLink =
+      Uri.parse('https://reportcontent.google.com/troubleshooter').replace(
+    queryParameters: {
+      'product': 'dart_pub',
+      'content_id': subject.fqn,
```

Comment:
`content_id` should be a canonical URL for the content in question.

One of:

* `pub.dev/publishers/<publisher>`
* `pub.dev/packages/<package>`
* `pub.dev/packages/<package>/versions/<version>`

```suggestion
      'content_id': subject.fqn, // TODO: Use subject.canonicalUrl, also add canonicalUrl
```